<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerSettings</class>
 <widget class="QWidget" name="LayerSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_13">
     <item row="1" column="1" rowspan="6">
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Preview</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_12">
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="demoLayout"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Show Label</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="1">
           <widget class="QCheckBox" name="xLabelCheckBox">
            <property name="text">
             <string>X Label</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="yLabelCheckBox">
            <property name="text">
             <string>Y Label</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="showTitleCheck">
            <property name="text">
             <string>Title</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QGroupBox" name="groupBox_5">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Title</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="2">
           <widget class="GuiUtils::ColorPushButton" name="titleColorButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Title</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="titleTextEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Font</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="GuiUtils::FontPushButton" name="fontButton">
            <property name="text">
             <string>Font</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Transparency</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QSlider" name="titleOpacitySlider">
            <property name="value">
             <number>99</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" rowspan="4">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Layer</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="2" column="0">
         <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0,0,0" rowminimumheight="0,0,0,0,0,0,0">
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Gradient Fill</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QComboBox" name="gradientFillCombo">
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>One Color</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Two Colors</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lightnessLabel">
            <property name="text">
             <string>Lightness</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="GuiUtils::ColorPushButton" name="startColorButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QSlider" name="lightnessSlider">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="secondColorLabel">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="GuiUtils::ColorPushButton" name="stopColorButton">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="directionLabel">
            <property name="text">
             <string>Direction</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QComboBox" name="directionCombo"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Border Radius</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QSpinBox" name="borderRadiusSpinBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GuiUtils::ColorPushButton</class>
   <extends>QPushButton</extends>
   <header>comguiutils/colorpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>GuiUtils::FontPushButton</class>
   <extends>QPushButton</extends>
   <header>comguiutils/fontpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
